quantum/quantum.h

```diff
@@ -267,9 +257,9 @@ void breathing_period_default(void);
 void breathing_period_set(uint8_t value);
 void breathing_period_inc(void);
 void breathing_period_dec(void);
+    #endif
```

Comment:
```suggestion
#endif
```
preprocessor blocks should stay left aligned imo (opinion)